pkg/controller/user-cluster-controller-manager/resources/resources/node-local-dns/deamonset.go

```diff
@@ -92,7 +91,7 @@ func DaemonSetCreator(registryWithOverwrite registry.WithOverwriteFunc) reconcil
 					ImagePullPolicy: corev1.PullAlways,
 					Args: []string{
 						"-localip",
-						CacheAddress,
+						machinecontroller.NodeLocalDNSCacheAddress,
```

Comment:
I meant rename `CacheAddress` to `NodeLocalDNSCacheAddress`. 

Importing `"k8c.io/kubermatic/v2/pkg/resources/machinecontroller"` introduces dependency on the the machine controller.